server/portal/apps/tickets/rtUtil.py

```diff
@@ -16,10 +16,10 @@ def __init__(self):
 
     def getUserTickets(self, userEmail, status="ALL"):
         if not status == "ALL":
-            ticket_list = self.tracker.search(Queue=rt.ALL_QUEUES, Requestors__exact=userEmail, Status__exact=status,
+            ticket_list = self.tracker.search(Queue=rt.ALL_QUEUES, Requestor__exact=userEmail, Status__exact=status,
```

Comment:
Is this a confirmed needed change as well?

Reply:
Based on my conversation with Chandra, he mentioned making this change anyhow.
![image](https://github.com/user-attachments/assets/25cba30d-9434-4787-90df-9069131b6098)

Reply:
Yes, but this is a different endpoint, `tracker.search`. Can you confirm the correct field is `Requestor__exact`?

Reply:
The results appear to be the same regardless of whether that line says `Requestor` or `Requestors`.

Reply:
If the results are the same, then we do not need to change, since nothing is broken. The issue from PR above is in create_ticket endpoint and not search endpoint.  The parameter to `create_ticket` is Requestor (as seen in [RT source code:](https://github.com/bestpractical/rt/blob/master/lib/RT/Ticket.pm#L269) ).

Reply:
I think this change still needs to be reverted

Reply:
Yep. I'll reverse these changes.